core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -51,19 +53,29 @@ angular.module('oppia').directive('imageUploader', [
             return 'This file is not recognized as an image.';
           }
 
-          if (!file.type.match('image/jpeg') &&
-            !file.type.match('image/gif') &&
-            !file.type.match('image/jpg') &&
-            !file.type.match('image/png') &&
-            !file.type.match('image/svg\\+xml')) {
-            return 'This image format is not supported.';
-          }
+          if (scope.allowedImageType === 'svg') {
```

Comment:
Why are these changes required?

Reply:
Only svgs should be uploaded to the editor. Since I am reusing this directive I restricted it to only svg and avoiding png, and other formats.

Reply:
Not sure about the pattern here -- it seems like context is being leaked. @seanlip do you think this is acceptable in this case?

Reply:
In what way is context being leaked? It seems that a reasonable way to configure an "image uploader" is to specify what image types are allowed.

However, if you want to do this, you should do it fully: pass in allowedImageTypes everywhere and have it represent a list. Maybe declare a constant list in constants.ts or somewhere and use that for the majority of the editors.

Reply:
I have changed it everywhere PTAL.